src/Microsoft.ML.Legacy/AssemblyRegistration.cs

```diff
@@ -38,10 +37,9 @@ public static void RegisterAssemblies(IHostEnvironment environment)
         /// </summary>
         private static bool LoadStandardAssemblies()
         {
-            Assembly apiAssembly = typeof(LambdaTransform).Assembly; // ML.Api
-            AssemblyName apiAssemblyName = apiAssembly.GetName();
+            Assembly dataAssembly = typeof(TextLoader).Assembly; // ML.Data
+            AssemblyName dataAssemblyName = dataAssembly.GetName();
 
-            _ = typeof(TextLoader).Assembly; // ML.Data
             //_ = typeof(EnsemblePredictor).Assembly); // ML.Ensemble BUG https://github.com/dotnet/machinelearning/issues/1078 Ensemble isn't in a NuGet package
```

Comment:
While you are in here - can you uncomment this line and remove the "BUG" comment?